test/REQUIRE

```diff
@@ -4,3 +4,4 @@ ParameterizedFunctions 3.0.0
 StaticArrays
 DiffEqCallbacks
 DiffEqOperators
+Compat
```

Comment:
where is this used?

Reply:
iterator_test.jl

Reply:
https://github.com/JuliaDiffEq/OrdinaryDiffEq.jl/commit/6eeabcb4024c054718df761cd2ce03ecf4b02302